client/checkout/classic/upe.js

```diff
@@ -170,11 +170,24 @@ jQuery( function ( $ ) {
 	 */
 	const getSelectedGatewayPaymentMethod = () => {
 		const gatewayCardId = getUPEConfig( 'gatewayId' );
-		const selectedGatewayId = $(
-			'li.wc_payment_method input.input-radio:checked'
-		).attr( 'id' );
+		let selectedGatewayId = null;
+
+		if ( $( 'li.wc_payment_method' ).length ) {
+			selectedGatewayId = $(
+				'li.wc_payment_method input.input-radio:checked'
+			).attr( 'id' );
+		} else if ( $( 'li.woocommerce-PaymentMethod' ).length ) {
```

Comment:
Ah I see, this is used inside http://localhost:8082/my-account/add-payment-method/. I was trying to look for this on the http://localhost:8082/checkout/ but I can't find it. 

I wonder if it's worth adding a comment here so we don't accidentally remove it in the future.

Reply:
Yes! This was a head-scratcher for me as well. [Comment added](https://github.com/Automattic/woocommerce-payments/pull/4427/commits/a5e6b957163c012a190fbbf7704dc1dafe67a555).